code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -50,6 +50,8 @@ Difficulty: Hard
 	idle_vision_range = 5
 	loot = list(/obj/structure/closet/crate/necropolis/bubblegum)
 	var/charging = 0
+	medal_type = "Bubblegum"
```

Comment:
Should I make a define for every one? I have to use strings with exact wording because of how medals work.